lib/stream-chat/stream_response.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# lib/stream_response.rb
+require 'stream-chat/stream_rate_limits'
+
+module StreamChat
+  class StreamResponse < Hash
+    attr_reader :rate_limit
+    attr_reader :status_code
+    attr_reader :headers
+
+    def initialize(hash, response)
+      super(nil)
+      merge!(hash)
+
+      if response.headers.key?('X-Ratelimit-Limit') # rubocop:disable Style/GuardClause
```

Comment:
Rubocop does not point this as an error when I run `bundle exec rubocop` locally. Am I missing something?

I just want to understand why we need the comment here.

Reply:
do you have latest rubocop locally? CI installs fresh everytime

Reply:
this lint ignore was for a previous version of this code (where rate_limit was a dictionary, not a class). now it works fine without it, so I just removed it.

Reply:
Superb, thanks!

Reply:
Awesome 😄